src/router/routes/api/draftmancer/publish.ts

```diff
@@ -122,103 +79,25 @@ export const handler = async (req: Request, res: Response) => {
       players: [],
     };
 
+    let drafterName: string = '';
     for (const player of publishDraftBody.players) {
-      const draftmancerPicks: DraftmancerPick[] = [];
       let mainboard: number[][][] = setupPicks(2, 8);
       let sideboard: number[][][] = setupPicks(1, 8);
-      const pickorder: number[] = [];
-      const trashorder: number[] = [];
-
-      for (const pick of player.picks) {
```

Comment:
Do to all the loops and calls to functions that would have to be mocked like getReasonableCardByOracle, I extracted bits whole into the draftmancerUtils, which both simplified testing the extract bits but also this endpoint